packages/plugins/docs-ui/src/lib/components/links/document-attach-dialog.component.scss

```
@@ -0,0 +1,62 @@
+// Self-contained: this dialog is standalone and is opened from app pages where
+// `DocsUiModule`'s component styles are not loaded, so it cannot inherit the
```

Stylelint (17.14.0)

[error] Unclosed string (CssSyntaxError)

(CssSyntaxError)